packages/base-styles/_mixins.scss

```diff
@@ -445,6 +445,9 @@
 	--wp-admin-theme-color-darker-10: #{darken($color-primary, 5%)};
 	--wp-admin-theme-color-darker-20: #{darken($color-primary, 10%)};
 
+	// Lighter shades.
```

Comment:
Is there any way we can use a lighter shade without adding a new CSS variable?

A sass variable (admin theme aware of course) would be a step in the right direction. But it would be even better if you could use the existing theme color variable, just in a lighter shade. Could you use layered CSS gradients to overlay a sheet of semi transparent white on the css variable background?

The problem with a new CSS variable is that once you've added it, people will start to use it for other things, which is dubious at best.

Reply:
Oooh, the multiple gradient is a nice trick!

My dev mindset still would rather have a nice variable shared to everyone, but still, it's an easy change 😄